schema/public.openapi.yaml

```diff
@@ -68,6 +68,26 @@ paths:
           $ref: '#/components/responses/BadRequest'
         '403':
           $ref: '#/components/responses/Forbidden'
+  
+  /internal/version:
```

Comment:
As this is an internal resource it should be defined in private.openapi.yaml instead